composer.json

```
@@ -16,20 +16,19 @@
         }
     ],
     "require": {
-        "php": ">=7.3",
-        "mybuilder/cronos": "~3.0",
-        "doctrine/annotations": "1.*",
-        "symfony/console": "^4.4 || ^5.0 || ^6.0",
-        "symfony/config": "^4.4 || ^5.0 || ^6.0",
-        "symfony/dependency-injection": "^4.4 || ^5.0 || ^6.0",
-        "symfony/framework-bundle": "^4.4 || ^5.0 || ^6.0",
-        "symfony/http-kernel": "^4.4 || ^5.0 || ^6.0",
-        "symfony/property-access": "^4.4 || ^5.0 || ^6.0",
-        "symfony/yaml": "^4.4 || ^5.0 || ^6.0"
+        "php": ">=8.0.2",
+        "mybuilder/cronos": "^3.0 || ^4.0",
+        "doctrine/annotations": "^1.14 || ^2.0",
```

Could we leave this as any version of 1 as it was previously?
```suggestion
        "doctrine/annotations": "1.* || ^2.0",
```

---

I am not sure what exact version we need 1.14.
I removed support for version 2. We can not hove both in my eyes at the moment.
I have added an extra testrun for --prefer-lowest. That is good to verify the lower bounds.

Failing with --prefer-lowset  https://github.com/Chris53897/cronos-bundle/actions/runs/3849825039/jobs/6559262106
Working with `1.*`
https://github.com/Chris53897/cronos-bundle/actions/runs/3849837894

If you want you can investigate what exact version is needed.

---

@maxbaldanza Testrun fails with "^1.7"
https://github.com/Chris53897/cronos-bundle/actions/runs/3854405905/jobs/6568364227